tanner/create_tables.py

```diff
@@ -0,0 +1,71 @@
+import asyncio
+
+
+class CreateTables:
+    @staticmethod
+    async def create_data_tables(pg_client):
+        """Create all the required tables in
+            the postgres database
+
+        Arguments:
+            pg_client {aiopg.pool.Pool}
+        """
+        async with pg_client.acquire() as conn:
+            async with conn.cursor() as cur:
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "paths" (
+                    "sess_uuid" UUID PRIMARY KEY NOT NULL UNIQUE,
+                    "path" TEXT NOT NULL,
+                    "timestamp" TIMESTAMP NOT NULL,
```

Comment:
Here you are using a name which duplicates the type, use the opportunity to describe the purpose of this field. I assume something like `create_at` maybe? You can also give it a default value like `now()` so you don't have to create it client side and let the database populate it for you.